### PR TITLE
Feature/seo

### PR DESCRIPTION
**Pull request by alexmonteiroqccareer**
Reviews Meta Description on 11 pages

### CODE REVIEW

File: src/components/seo.tsx [hunk 1]
```diff
@@ -5,9 +5,10 @@ import React from 'react';
 interface Props {
   title: string;
   description: string;
+  canonical?: string;
```
Comment (davewelsh) [inline src/components/seo.tsx]: Didn't you say you wanted all pages to have a canonical meta tag? Then we shouldn't have this prop be optional

Comment (davewelsh) [reply src/components/seo.tsx]: Test

Comment (davewelsh) [reply src/components/seo.tsx]: test

Comment (davewelsh) [reply src/components/seo.tsx]: @alexmonteiroqccareer

Comment (alexmonteiroqccareer) [reply src/components/seo.tsx]: @davewelsh  ok. I will fix it.

**Comment by davewelsh** [comment]
@alexmonteiroqccareer Please revise

**Pull request closed without merging**